cypress/integration/orders/orders.PoSearch.spec.js

```diff
@@ -48,4 +49,16 @@ describe('orders: Test PO search', () => {
         Orders.checkSearchResults(orderNumber);
       });
   });
+  it('C6718 Test the PO filters [except tags]', { tags: [TestType.smoke] }, () => {
```

Comment:
I suggest to define list of filters and run forEach with parametrized it() into each iteration